### DISCUSSION TITLE
Windows Installation

### DISCUSSION THREAD

**Discussion by Bradladam** [junzis/pyModeS#104] [Q&A]
I was wondering if it's possible to install and use this on windows. 
I've installed PyModeS and Pyrtlsdr and python3, but I'm not sure how to get modeslive working, or if it's possible to get working on windows. When I try to run modeslive in the system console it said 'modeslive' is not recognized as an internal or external command, operable program or batch file.
I thought that maybe it doesn't matter and that's just to see the live traffic view, so I tried to run the python commands using
import pymodes as pms, but they all say 'msg' is not defined.

I'm very new to these things so I apologise for my ignorance

Thank you in advance :) also big fan

**Comment by junzis** [comment]
I doubt modeslive will work for Windows since it uses a Linux terminal. Maybe you can try the "Windows subsystem for Linux". I believe it is something you can "install" on Windows. ( I haven't used windows for more than 10 years... :man_facepalming: )

**Reply by xoolive** [reply]
For Windows, you may `pip install windows-curses`.
@junzis that's Section 21.2 of you-know-what 😜
